.cursor/skills/teapie/scripts/insert-gap.py

```
@@ -0,0 +1,279 @@
+        gap_size: Size of gap to create (default: 1)
+    """
+    if not directory.exists():
+        print(f"❌ Error: Directory does not exist: {directory}")
+        sys.exit(1)
+    
+    if not directory.is_dir():
+        print(f"❌ Error: Path is not a directory: {directory}")
+        sys.exit(1)
+    
+    if gap_size < 1:
+        print(f"❌ Error: Gap size must be at least 1")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)